lib/swagger-mock-validator/validate-spec-and-mock/validate-parsed-mock-response-body.ts

```diff
@@ -68,7 +68,7 @@ export const validateParsedMockResponseBody = (
     return [];
   }
 
-  const expectedMediaType = parsedMockInteraction.requestHeaders.accept?.value;
+  const expectedMediaType = parsedMockInteraction.requestHeaders.accept?.value || "application/json";
```

Comment:
Is the idea here that we default to JSON in the case no accept header is defined or found?

Reply:
Yes. Previously this defaulting happened elsewhere. But I've relocated it here.